sdks/android/Mentat/library/src/main/java/com/mozilla/mentat/InProgressTransactionResult.java

```diff
@@ -41,7 +41,7 @@ public InProgress getInProgress() {
     
     public TxReport getReport() {
         if (this.result.isFailure()) {
-            Log.e("InProgressTransactionResult", this.result.err);
+            Log.e("InProgressTransactRes", this.result.err);
```

Comment:
What was the reason for this log change?

Reply:
Android log tags are limited to 23 characters in length. The original tag was 28.